tests/d3d12_device.c

```diff
@@ -1506,3 +1506,39 @@ void test_enumerate_meta_commands(void)
     refcount = ID3D12Device5_Release(device5);
     ok(!refcount, "ID3D12Device has %u references left.\n", (unsigned int)refcount);
 }
+
+void test_vtable_origins(void)
+{
+#ifdef _WIN32
+    HMODULE d3d12core_module, vtable_module;
+    MEMORY_BASIC_INFORMATION info;
+    ID3D12Device *device;
+    SIZE_T ret;
+
+    /* Skip this test if running on an older version of 
+     * Windows from before the D3D12 + D3D12Core split. */
+    if (!(d3d12core_module = LoadLibraryA("d3d12core")))
```

Comment:
It's more appropriate to GetModuleHandle after creating the d3d12 device I think. d3d12 is supposed to load it on its own.